calamari-common/calamari_common/remote/mon_remote.py

```diff
@@ -1057,3 +1065,30 @@ def tail(subpath, n_lines):
     p = subprocess.Popen(["tail", "-n", str(n_lines), path], stdout=subprocess.PIPE)
     stdout, stderr = p.communicate()
     return stdout
+
+
+def cluster_stats(cluster_name):
+    import rados
+
+    cluster = rados.Rados(name=RADOS_NAME, clustername=cluster_name, conffile='')
+    cluster.connect()
+
+    result = cluster.get_cluster_stats()
+
+    return result
+
+
+def pool_stats(cluster_name, pool_id):
```

Comment:
@b-ranto I'd like to building up of a data structure that could be used to support a list-view here. so maybe iterate over http://docs.ceph.com/docs/master/rados/api/python/#rados.Rados.list_pools and return a dict of pool name: stats